ibm_catalog.json

```
@@ -0,0 +1,184 @@
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-scc-workload-protection/blob/main/solutions/instances/README.md",
+        "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-scc-workload-protection/main/images/scc-icon.svg",
+        "provider_name": "IBM",
+        "features": [
+          {
+            "title": "Security and Compliance Center Workload Protection",
+            "description": "Creates and configures a Security and Compliance Center Workload Protection instance."
+          }
+        ],
+        "support_details": "This product is in the community registry, as such support is handled through the originated repo. If you experience issues please open an issue in that repository [https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/issues](https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/issues). Please note this product is not supported via the IBM Cloud Support Center.",
+        "flavors": [
+          {
+            "label": "Workload Protection",
+            "name": "workload-protection",
+            "install_type": "extension",
+            "working_directory": "solutions/standard",
+            "compliance": {
+              "authority": "scc-v3",
+              "profiles": [
+                {
+                  "profile_name": "IBM Cloud Framework for Financial Services",
+                  "profile_version": "1.7.0"
+                }
+              ]
+            },
+            "configuration": [
+              {
+                "key": "ibmcloud_api_key",
+                "displayname": "API KEY"
+              },
+              {
+                "key": "provider_visibility",
+                "displayname": "Provider Visibility",
+                "options": [
+                  {
+                    "displayname": "private",
+                    "value": "private"
+                  },
+                  {
+                    "displayname": "public",
+                    "value": "public"
+                  },
+                  {
+                    "displayname": "public-and-private",
+                    "value": "public-and-private"
+                  }
+                ]
+              },
+              {
+                "key": "use_existing_resource_group",
+                "displayname": "Use Existing Resource Group",
+                "required": true
+              },
+              {
+                "key": "resource_group_name",
+                "displayname": "Resource Group Name",
+                "required": true
+              },
+              {
+                "key": "prefix",
+                "displayname": "Prefix",
+                "required": true,
+                "description": "The prefix to add to all resources that this solution creates. To not use any prefix value, you can enter the string `__NULL__`."
```

New guidance for prefix: https://github.ibm.com/GoldenEye/issues/issues/12841
```suggestion
                "description": "The prefix to add to all resources that this solution creates (e.g `prod`, `test`, `dev`). To not use any prefix value, you can enter the string `__NULL__`."
```